test/smoke/healthchecker/src/test/test.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+/* eslint-disable no-unused-expressions */
+
+const request = require('abacus-request');
+const debug = require('abacus-debug')('abacus-healthchecker-smoke-test');
+
+const testEnv = {
+  healthchecker: process.env.HEALTHCHECKER_URL || 'http://localhost:9884',
+  startTimeout: process.env.SMOKE_START_TIMEOUT || 10000,
+  totalTimeout: process.env.SMOKE_TOTAL_TIMEOUT || 60000,
+  clientId: process.env.HYSTRIX_CLIENT_ID,
+  clientSecret: process.env.HYSTRIX_CLIENT_SECRET,
+  prefix: process.env.ABACUS_PREFIX || ''
+};
+
+const authentication = {
+  auth: {
+    user: testEnv.clientId,
+    pass: testEnv.clientSecret,
+    sendImmediately: true
+  }
+};
+
+describe('abacus-healthchecker-smoke-test', () => {
+  before((done) =>
+    request.waitFor(`${testEnv.healthchecker}/v1/healthcheck`, authentication, testEnv.startTimeout, done)
+  );
+
+  const check = (alias, groups, done) => {
+    request.get(`${testEnv.healthchecker}${alias}`, authentication, (err, response) => {
+      debug('Healthchecker err %o, response %j', err, response);
+
+      expect(err).to.be.undefined;
+      expect(response.statusCode).to.equal(200);
+      expect(response.body).to.be.an('object').that.has.all.keys(groups);
+
+      for(let appGroup of Object.keys(response.body)) {
+        const appsResponses = Object.values(response.body[appGroup]);
+        appsResponses.every((responseCode) => expect(responseCode, appGroup + ': ' + responseCode).to.equal(200));
+      }
+      done();
+    });
+  };
+
+
+  it('reports the health of client facing components', function(done) {
+    this.timeout(testEnv.totalTimeout + 2000);
+    const testGroups = [
```

Comment:
In the `healthchecker` the internal and external applications are configurable. I would expect that this is the same in the `smoke` test as well so that the smoke test can be used on landscapes with various deployment schemes.